### PR TITLE
File Added : Assgn5 and Assgn6

### PR DESCRIPTION
**Pull request by yogesh545**

### CODE REVIEW

**Review by Jestinxavier — APPROVED** [review]

**Review by Priyankashelke2411 — APPROVED** [review]

File: yogesh_khande/html/assignment 5/website.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                color: rgba(235, 229, 229, 0.966);
+            }
+            p {
+                color: black;
+                background-color: hsla(9, 100%, 64%, 1);
+            }
+            .solid {
+                border-style: solid;
+            }
+            p {
```
Comment (KomalShinde813) [inline yogesh_khande/html/assignment 5/website.html]: duplicate style for p tag

File: yogesh_khande/html/assignment 5/website.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                color: black;
+                background-color: hsla(9, 100%, 64%, 1);
+                margin: 5px;
+                padding: 10px;
+            }
+            .solid {
```
Comment (KomalShinde813) [inline yogesh_khande/html/assignment 5/website.html:23]: duplicate style

File: yogesh_khande/html/assignment 5/website.html [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+            <br/>
+            <br/>
+            <br/>
+            <h2 class="h2">More About Me</h2>
+            <br/>
+            <br>
+            <br>
+            <div>
+            <p class="solid">
+                <a href="resume.html" target="_blank">My Resume</a><br/>
```
Comment (KomalShinde813) [inline yogesh_khande/html/assignment 5/website.html]: where is html files which are mentioned in href

File: yogesh_khande/html/assignment6/form.html [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+                background-color: rgb(83, 120, 177);
+                text-align-last: right;
+                border-radius: 2px 8px;
+            }
+        </style>
+    </head>
+    <body>
+        <h1 class="h1">MI-11 Intern HTML Assesment</h1>
+        <br>
+        <div>
```
Comment (KomalShinde813) [inline yogesh_khande/html/assignment6/form.html]: why you use extra div here

**Review by KomalShinde813 — CHANGES_REQUESTED** [review]

**Review by KomalShinde813 — APPROVED** [review]

**Review by vdongre304 — APPROVED** [review]

**Pull request merged**